### ISSUE TITLE
Is there a way to not log every object that gets parsed?

### ISSUE THREAD

**Issue by anchpop** [Seanba/Tiled2Unity#118]
Sometimes I have a ton of objects and it Tiled2Unity becomes quite slow. Is there any way (maybe a command line argument?) to disable "verbose" logging like:

```
Parsing object ...
Parse properites ...
[repeat ad nauseum]
```

Also, as someone with dyslexia, I can relate to having misspelled "properties" once or twice :)

**Comment by Seanba** [comment]
Hi there. The next version of Tiled2Unity will greatly reduce the amount of information displayed unless you use the -V (or --verbose) command line option. I should have it ready in the next day or too.

**Comment by Seanba** [comment]
I just published version 1.0.12.4 of Tiled2Unity which contains this fix. Opening TMX files and exporting should be considerably faster for you now.

**Comment by anchpop** [comment]
Thank you!
